ui-tests/src/main/java/org/jenkins/plugins/pitmutation/WebElementUtils.java

```diff
@@ -0,0 +1,66 @@
+package org.jenkins.plugins.pitmutation;
+
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+import static org.jenkinsci.test.acceptance.Matchers.by;
+
+/**
+ * Represents utils for WebElements.
+ */
```

Comment:
I think inlining that class would make the code more readable (at least for someone already familiar with ATH)